packages/react-native-reanimated/src/createAnimatedComponent/NativeEventsManager.ts

```diff
@@ -0,0 +1,144 @@
+'use strict';
+import type {
+  INativeEventsManager,
+  IAnimatedComponentInternal,
+  AnimatedComponentProps,
+  InitialComponentProps,
+  AnimatedComponentRef,
+} from './commonTypes';
+import { has } from './utils';
+import { WorkletEventHandler } from '../WorkletEventHandler';
+import { findNodeHandle } from 'react-native';
+
+export class NativeEventsManager implements INativeEventsManager {
+  private _managedComponent: ManagedAnimatedComponent;
+  private _componentOptions?: ComponentOptions;
+  private _eventViewTag = -1;
```

Comment:
Me thinks

```suggestion
  private #managedComponent: ManagedAnimatedComponent;
  private #componentOptions?: ComponentOptions;
  private #eventViewTag = -1;
```

Reply:
There can't be both at once, `#` symbol is sufficient